### DISCUSSION TITLE
Custom drawer css

### DISCUSSION THREAD

**Discussion by quangle36** [saadeghi/daisyui#2881] [General]
Hi everyone,
I want to change the breakpoint of drawer-open to another number. How can I achieve this?
<img width="492" alt="image" src="https://github.com/saadeghi/daisyui/assets/58833659/47aed8c4-e5dd-4d47-944b-c3feff75e126">

**Comment by saadeghi** [comment]
You used `lg:drawer-open`  
You can use another breakpoint prefix: https://tailwindcss.com/docs/responsive-design
